src/test/functional/data.ts

```diff
@@ -1,5 +1,6 @@
 export const userTestData = {
```

Comment:
### This is just a suggestion for easier maintenance and readability in the future.
You have a const here named userTestData - which indicates that this dictionary stores normal user data? But after that we can see that there is also admin data in the dictionary. My suggestion if you would like to do it like this - split it into two separate dictonaries. First for normal user and second for admin user. After that you can call the whole user dictionary in tests. Something like this?:
```
export const normalUserTestData = {
  password: "passwordNormalUser",
  userName: "Normal User"
};

export const adminUserTestData = {
  password: "passwordAdminUser",
  userName: "Admin User"
};
```
And then in test you can call it as single parameter not two.
`await loginPage.loginUser(testData.adminUserTestData)`

Reply:
right i will improve it